interfaces/webrtc.idl

```diff
@@ -0,0 +1,657 @@
+dictionary RTCConfiguration {
```

Comment:
There will need to be a webrtc-stats.idl for https://w3c.github.io/webrtc-stats/ eventually, so can you name this file webrtc-pc.idl?